mobile/src/features/ReviewTx/useCases/ReviewTxScreen/ReviewTx/Overview/OverviewTab.tsx

```diff
@@ -730,6 +730,7 @@ const useShowOperationsNotice = (operations: Operations) => {
   const {openModal} = useModal()
   const strings = useStrings()
   const screenHeight = useWindowDimensions().height
+  const hasOpenedInSession = React.useRef(false)
```

Comment:
this is not needed. It needs to be able to update state after the modal is shown and read the value update so the modal its now shown anymore.